packages/deploy-tooling/README.md

```diff
@@ -57,6 +57,7 @@ Options:
   --verbose                            verbose log output (default: false)
   --destination  <destination>         Destination in SAP BTP pointing to an ABAP system
   --url <target-url>                   URL of target ABAP system
+  --service                            Target SAPUI5 Respository OData Service
```

Comment:
Could we indicated that this is to be used for an optional alias for the service and usually not required?

Reply:
Done.